dev-python/influxdb/influxdb-5.3.0-r1.ebuild

```diff
@@ -30,6 +30,8 @@ BDEPEND="test? (
 		dev-python/requests-mock[${PYTHON_USEDEP}]
 	)"
 
+PATCHES=( "${FILESDIR}/${PV}-pandas-future-warning.patch" )
```

Comment:
Normally we prefix name files as: ${PN}-VERSION-DESCRIPTION.patch, where VERSION is the version it was based on.

We will use ${P} in the ebuild if it's an upstreamed patch (it's just a little hint), but ${PN}-5.3.0-foo.patch for example if it's not upstreamed but based on 5.3.0.

Reply:
Done.